icicle/tests/test_base.h

```diff
@@ -53,7 +49,13 @@ class IcicleTestBase : public ::testing::Test
   static void TearDownTestSuite() {}
 
   // SetUp/TearDown are called before and after each test
-  void SetUp() override {}
+  void SetUp() override
+  {
+    unsigned seed = time(NULL);
+    srand(seed);
```

Comment:
why do you need srand here?

Reply:
Leftover from before rand_gen.h - I'll remove it